core/ecschema-metadata/src/test/Metadata/Constant.test.ts

```diff
@@ -332,6 +332,99 @@ describe("Constant", () => {
       assert.strictEqual(constantSerialization.definition, "PI");
       assert.strictEqual(constantSerialization.phenomenon, "TestSchema.TestPhenomenon");
     });
+
+    it("async - when only numerator property is defined", async () => {
```

Comment:
Test name should be clear that we are testing to make sure default values are not serialized

Reply:
Names are modified now.